pre_commit/languages/golang.py

```diff
@@ -21,6 +22,7 @@
 
 def get_env_patch(venv):
     return (
+        ('GOBIN', UNSET),
```

Comment:
is this necessary now? I'd expect this env var to not matter at runtime

Reply:
It doesn't appear to be necessary.  Presumably the new entry in $PATH should supercede whatever is in $GOBIN.